jwks/jwks.go

```diff
@@ -13,13 +13,13 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/ONSdigital/dp-identity-api/models"
+	"github.com/ONSdigital/dp-identity-api/v2/models"
 	dphttp "github.com/ONSdigital/dp-net/v2/http"
 )
 
 //go:generate moq -out mock/jwks.go -pkg mock . JWKSInt
-type JWKSInt interface {
-	JWKSGetKeyset(awsRegion, poolId string) (*JWKS, error)
+type JWKSInt interface { //nolint // JWKSInt serves the purpose of maintaining explicit, domain-specific naming that aligns with the rest of the system.
```

Comment:
Exported identifiers should have godoc comments.
Same for `JSONKey` and `JWKS` structs too.